src/main/java/com/openshift/restclient/model/image/IImageStreamImport.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.openshift.restclient.model.image;
+
+import java.util.Collection;
+
+import com.openshift.restclient.images.DockerImageURI;
+import com.openshift.restclient.model.IResource;
+import com.openshift.restclient.model.IStatus;
+
+/**
+ * 
+ * @author jeff.cantrill
+ *
+ */
+public interface IImageStreamImport extends IResource {
+
+	/**
+	 * Set to true to import tags; false to just retrieve
+	 * @param importTags
+	 */
+	void setImport(boolean importTags);
+
+	boolean isImport();
+
+	/**
+	 * Add image info those being imorted
+	 * @param string 
+	 * @param uriWithoutTag
+	 */
+	void addImage(String fromKind, DockerImageURI imageUri);
+
+	/**
+	 * The status of the image retrieval
+	 * @return
+	 */
+	Collection<IStatus> getImageStatus();
+	
+	/**
+	 * Get the raw json docker metadata for
+	 * the given uir.  Tries to match uri without tag
+	 * to the beginning of image.dockerImageReference
+	 * 
```

Comment:
The result of posting this resource grabs the tags and returns them to you.  This allows you to get the raw json to further inspect. I don't know if this content maps directly to docker structures and I didn't want to add a dependency to the linux docker tooling here.  Additionally, I dont know if it was worth trying to add a class to represent this info.  It would be really nice if we could get items without having to return a class; walking across arrays is still a problem